src/Network.cpp

```diff
@@ -862,9 +862,38 @@ Network::Netresult Network::get_scored_moves(
     if (ensemble == DIRECT) {
         assert(rotation >= 0 && rotation <= 7);
         result = get_scored_moves_internal(planes, rotation);
+    } else if (ensemble == MULTI_AVG && rotation > 1) {
+        // MULTI_AVG rotation is the # of random rotations to provide
+        int num_rotations = rotation;
```

Comment:
There doesn't seem to be a use case for averaging multiple (instead of all) rotations, so you can simplify the code by removing support for this. That also avoids the awkward overload of "rotation" having multiple meanings.